python/perf-kernels/flash-attention.py

```diff
@@ -209,27 +289,54 @@ def _attn_fwd(
     stride_vz, stride_vh, stride_vk, stride_vn,
     stride_oz, stride_oh, stride_om, stride_on,
     stride_bz, stride_bh, stride_bm, stride_bn,
-    extra_tokens_n,
-    H,
-    seqlen_q,
-    seqlen_k,
+    cu_seqlens_q_tensor, cu_seqlens_k_tensor,
     dropout_p,
     philox_seed,
     philox_offset_base,
     encoded_softmax,
+    max_seqlens_q, max_seqlens_k,
+    VARLEN: tl.constexpr,
+    B: tl.constexpr, HQ: tl.constexpr, HK: tl.constexpr,
     STAGE: tl.constexpr,
     BLOCK_M: tl.constexpr,
     BLOCK_DMODEL: tl.constexpr,
     BLOCK_N: tl.constexpr,
     PRE_LOAD_V: tl.constexpr,
-    NEED_PADDING: tl.constexpr,
     BIAS_TYPE: tl.constexpr,
     ENABLE_DROPOUT: tl.constexpr,
     RETURN_ENCODED_SOFTMAX: tl.constexpr
 ):
+    MQA: tl.constexpr = HQ != HK
     start_m = tl.program_id(0)
-    off_hz = tl.program_id(1)
-    q_offset = off_hz * stride_qh
+    off_h_q = tl.program_id(1)
+    off_z = tl.program_id(2)
+    if VARLEN:
+        cu_seqlens_q_start = tl.load(cu_seqlens_q_tensor + off_z)
```

Comment:
Can we load a size `[2]` Tensor with single `tl.load`?

Optional, can be done later since it's less relevant.